oidc-wp/openid-connect-generic
# Fix #178 : update user meta after refresh

Fix #178 

### All Submissions:


### Changes proposed in this Pull Request:

Update last-token-response user meta after refresh.

### How to test the changes in this Pull Request:

1. Connect with OpenID
2. Check access token (from user meta) against OpenID provider : OK
3. Wait for expiration / token refresh
4. Check again access token against OpenID provider : OK (it's the new refreshed token now in user meta)

### Other information:


### Changelog entry

Update last-token-response user meta after refresh
